scripts/create_genesis.sh

```diff
@@ -13,6 +13,12 @@ WASM_SEQUENCES=$((WASM_SEQUENCES + 1))
 INITIAL_HEIGHT=$(jq '.initial_height' $SNAPSHOT_INPUT)
 ENABLE_HEIGHT=$((INITIAL_HEIGHT + 2))
 
+jq '.app_state.wasm.contracts |= map(if .contract_info.label | test("^[ \\t\\r\\n]+|[ \\t\\r\\n]+$") then .contract_info.label |= gsub("^[ \\t\\r\\n]+|[ \\t\\r\\n]+$"; "") else . end)' $SNAPSHOT_INPUT > temp.json
+
+jq '.app_state.bank.denom_metadata |= map(select(.name == "Neutron"))' temp.json > temp1.json
```

Comment:
Have you tried to do this using jq pipes like few lines below? In this case it will process whole file in one run, instead of 3